src/rpc/conn_pool.rs

```diff
@@ -319,42 +316,42 @@ struct SharedPool {
     conn_builder: ConnBuilder,
     inner: Mutex<PoolInner>,
     cond: Condvar,
-    conn_init_thread_pool: Arc<ScheduledThreadPool>,
+    runtimes: RuntimesRef,
```

Comment:
Just provide a specific runtime is enough? I guess we can call it as bg_runtime?

Reply:
Fixed. We reduce the runtime (learn more from ObClientRuntimes), and use bg_runtime here.